### PR TITLE
Start adding tests and stuff

### PR DESCRIPTION
**Pull request by mlarraz**
Repo still needs to be added on Travis and Coveralls.

### CODE REVIEW

**Comment by jmmastey** [comment]
Added the repo on both services.

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```
Comment (jmmastey) [inline .rspec:2]: Probably makes sense to ignore this file, if anything.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 .DS_Store
+Gemfile.lock
```
Comment (jmmastey) [inline .gitignore:2]: I've never totally understood the ethic behind ignoring the lockfile, but I know that's a thing, and I'm willing to go with it. If so, can you rm the file that's tracked for now?

**Review by jmmastey — APPROVED** [review]

**Comment by jmmastey** [comment]
Hah. 44% with empty tests. There's some optimism.

**Comment by mlarraz** [comment]
Had to update to exclude the tests themselves, since they were boosting coverage

File: .gitignore (hunk 2 above)
Comment (mlarraz) [reply .gitignore:2]: For libraries, you can't guarantee many details about the environment they run in, particularly what version of a dependency is being used.

Ignoring the lockfile forces your tests to always run against the latest version of your stated dependencies, theoretically giving you an early warning that a new version has broken something.

Comment (jmmastey) [reply .gitignore:2]: Works for me.

**Pull request merged**